maxim/logger/langchain/tracer.py

```
@@ -799,6 +804,26 @@ def on_chain_error(
+                    if parent_container is not None:
+                        parent_container.end()
+                        if self.callback is not None:
+                            self.callback(
+                                "trace.ended",
+                                {
+                                    "trace_id": parent_container.id(),
+                                    "trace_name": parent_container.name(),
+                                },
+                            )
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)